### ISSUE TITLE
Plan of action! (so far)

### ISSUE THREAD

**Issue by jessimk** [UBC-MDS/DSCI-522-Jes-Alex#20]
@ehhope 

- [x] Prep script 1, load data (R)

- [x] Prep script 2, wrangling & write csv
-- [ ] add code to re-code null values
-- [ ] scriptify

- [x] Prep script 3, EDA & Decision Tree Analysis
-- [ ] EDA: .describe and feature histograms and discussion of findings
-- [ ] DT:  use all features (ideally), train_test_split (80%, 20%), plot K depth values for train and test accuracy, find top three predictors and plot feature prediction power and display tree.

- [x] Prep script 4, Report

- [ ] creating a gif bank

**Comment by ehhope** [comment]
Analysis script done
